src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Time.txt

```diff
@@ -88,17 +88,23 @@ true
 >> Time(1/0, 2, 3) >= Time(8,1,30)
 #Error
 
->> Text(Time(10, 70, 360))
-"11:16 AM"
+>> Text(Time(10, 70, 360), "HH:mm:ss")
+"11:16:00"
+
+>> Text(Time(10, -30, 360), "HH:mm:ss")
+"09:36:00"
 
->> Text(Time(10, -30, 360))
-"9:36 AM"
+>> Text(Time(10, 30, -360), "HH:mm:ss")
+"10:24:00"
 
->> Text(Time(10, 30, -360))
-"10:24 AM"
+>> Text(Time(10, -30, -360), "HH:mm:ss")
+"09:24:00"
```

Comment:
We should try Time(23, ...) and maybe with "hh:mm:ss" as well + other characters like YY, ffff, g, tt...

Reply:
I'll leave it to a future PR; what I wanted to do in this one was to make the tests more locale-independent (since "9:24 AM" is very US-centric). There is still more work to do in other test files

Reply:
But I'll add some sanity cases for larger times as well